src/Controllers/LoginController.php

```diff
@@ -39,7 +39,7 @@ public function loginAction()
             return redirect()->route('login')->withInput()->with('error', $result->reason());
         }
 
-        $user = $result->extraInfo();
+        $result->extraInfo();
```

Comment:
I think this line can be removed as it is not being used for anything

Reply:
Removed.